cmds/core/dhclient/dhclient.go

```diff
@@ -47,51 +47,79 @@ var (
 	v4Port = flag.Int("v4-port", dhcpv4.ServerPort, "DHCPv4 server port to send to")
 )
 
-func main() {
-	flag.Parse()
-	if len(flag.Args()) > 1 {
-		log.Fatalf("only one re")
-	}
+type opts struct {
+	timeout  int
+	retry    int
+	dryRun   bool
+	verbose  bool
+	vverbose bool
+	ipv4     bool
+	ipv6     bool
+	v6Port   int
+	v6Server string
+	v4Port   int
+}
 
-	if len(flag.Args()) > 0 {
-		ifName = flag.Args()[0]
+func run(args []string, opts opts) error {
```

Comment:
switch this up? 
func run(opts opts, args ...string) 
then, callers have an easier time of it maybe.

Reply:
Fixed in the latest commit.